packages/demo-app/src/examples/Examples.tsx

```diff
@@ -80,6 +81,7 @@ const ExamplesContainer: React.FC<{ isDark?: boolean }> = ({ isDark = false }) =
             <NonIdealStateExample />
             <PopoverExample />
             <SliderExample />
+            <SpinnerExample />
```

Comment:
I noticed that we didn't have demo examples for Spinner, and I wanted to test the validation warnings locally, so I added some examples.